packages/driver/src/cy/commands/screenshot.js

```diff
@@ -347,6 +347,10 @@ const takeScreenshot = (Cypress, state, screenshotConfig, options = {}) => {
 
   return before()
   .then(() => {
+    const win = state('window')
+
+    win.dispatchEvent(new CustomEvent('scroll'))
```

Comment:
Can you add a code comment here above this line referencing the issue and also describe (in the code comment) _why_ this is necessary / how it fixes the issue? I still don't understand that myself and a future developer won't either.